src/app/components/OptimoPromos/Image/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { string, number } from 'prop-types';
+import ImageWithPlaceholder from '#app/containers/ImageWithPlaceholder';
+import StyledImage from './index.styles';
+
+const Image = ({ src, altText, srcset, fallbackSrcset, width, height }) => {
+  const RATIO = 56.25;
```

Comment:
this is more obvious that the image aspect ratio is 16/9

```suggestion
const Image = ({ src, altText, srcset, fallbackSrcset, width, height }) => {
  const ASPECT_RATIO = 9 / 16 * 100;
```

Reply:
Done